src/gstreamer/gstkvssink.cpp

```diff
@@ -122,6 +122,10 @@ GST_DEBUG_CATEGORY_STATIC (gst_kvs_sink_debug);
 #define KVS_ADD_METADATA_VALUE "value"
 #define KVS_ADD_METADATA_PERSISTENT "persist"
 
+#define KVS_ADD_EVENT_METADATA_G_STRUCT_NAME "kvs-add-event-metadata"
+#define KVS_ADD_EVENT_METADATA_EVENT "event"
```

Comment:
Does gst-inspect-1.0 kvssink print info / actions about  new capability?

Reply:
It doesn't because this feature cannot be used from the command line because the function needs to be configured